isic_kredit/ViewControllers/CreditViewController.swift

```diff
@@ -104,133 +102,29 @@ class CreditViewController: UIViewController {
-                                                
-                                                let parameters1: Parameters = [
-                                                    input1 : input1Val,
-                                                    input2 : input2Val
-                                                ]
-                                                
-                                                
-                                                Alamofire.request(linkAction, method: .post, parameters: parameters1) .responseString { responseFinalSite in
-                                                    
-                                                    if let responseFinalSite = responseFinalSite.result.value{
-                                                        self.setMoneyValue(siteResultValue: responseFinalSite)
-                                                        self.hudSuccess(text: NSLocalizedString("Loaded", comment: ""))
-                                                    } else {
-                                                        self.hudError(text: NSLocalizedString("Fail", comment: ""))
-                                                        return
-                                                    }
-                                            
-                                                }
-                                                
-                                            } catch{
-                                                self.hudError(text: NSLocalizedString("Fail", comment: ""))
-                                                return
-                                            }
-                                            
-                                        }
-                                    }
-                                    
-                                }
-                            }
-                            
-                        }
-                    } else {
-                        self.hudError(text: NSLocalizedString("Fail", comment: ""))
-                        return
-                    }
-                    
-                  //User is logged in
-                } else {
-                    self.setMoneyValue(siteResultValue: (agataResponse.result.value)!)
-                    self.hudSuccess(text: NSLocalizedString("Loaded", comment: ""))
-                }
-                //There is something wrong with the site
-            } else {
-                self.hudError(text: NSLocalizedString("Fail", comment: ""))
-                return
-                
-            }
-            
-        }
+        dataManager.finalData(success: { response in
+                                self.setMoneyValue(value: response)
+                                 self.hudSuccess(text: NSLocalizedString("Loaded", comment: ""))
```

Comment:
Nechceme žádnej hud success, user chce vidět číslo a nezajímá ho, že se něco načetlo. Zmizelo loading kolečko, tak jasně že se to načetlo.